ddtrace/settings/exception_replay.py

```diff
@@ -1,16 +1,17 @@
 from envier import En
 
 
-class ExceptionDebuggingConfig(En):
-    __prefix__ = "dd.exception_debugging"
+class ExceptionReplayConfig(En):
+    __prefix__ = "dd.exception"
 
     enabled = En.v(
         bool,
-        "enabled",
+        "replay.enabled",
         default=False,
         help_type="Boolean",
         help="Enable automatic capturing of exception debugging information",
+        deprecations=[("debugging.enabled", None, "3.0")],
```

Comment:
Does this also log the deprecation warning to the customer + allow them to use the legacy env var until 3.0 is released?

Reply:
Yep 🙂